azure_monitor/simple_sample/readme.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Sample: Simple Python Applications with OpenCensus'
+description: Simple Python Applications with OpenCensus.
+author: givenscj
+ms.author: givenscj
+ms.devlang: python
+ms.topic: sample
+ms.date: 07/27/2022
+ms.custom: []
+---
+
+# Sample: Simple Python Applications with OpenCensus
```

Comment:
I believe that while there are some content in `simple_sample` that are useful, we would like to keep the surface and number of samples to be smaller and more distinct to represent actual Python customer usage. IMO we should remove the `simple_sample` folder and its contents.

Reply:
The docs don't have the setup aspects included to get samples to run or debug any issues hence having it exist at all.  I did delete a couple of the non-referenced examples and one that didn't really do anything.  Other part is the screen shots showing how to actually query for the app insights data for the samples (that doesn't really exist anywhere).